### PR TITLE
Settings help info

### PR DESCRIPTION
**Pull request by phunkyfish**
v3.22.0
- Added: Help info for addon settings
- Added: Delete child timers when deleting autotimers
- Added: Set max connection check interval to 60 seconds
- Fixed: Incorrect localisation IDs
- Fixed: Timers in error state cause crash on delete
- Added: Support show info fields for Timers

### CODE REVIEW

**Comment by phunkyfish** [comment]
@ksooo added an extra commit to get it ready for solution for https://github.com/xbmc/xbmc/issues/15606

File: src/enigma2/Timers.cpp [hunk 1]
```diff
@@ -978,13 +978,29 @@ PVR_ERROR Timers::DeleteAutoTimer(const PVR_TIMER &timer)
   {
     AutoTimer timerToDelete = *it;
 
+    //remove any child timers
+    bool childTimerIsRecording = false;
+    for (auto &childTimer : m_timers)
```
Comment (ksooo) [inline src/enigma2/Timers.cpp]: `const auto&` ?

**Review by ksooo — COMMENTED** [review]
minors only

File: src/enigma2/Timers.cpp [hunk 2]
```diff
@@ -941,24 +941,30 @@ PVR_ERROR Timers::DeleteTimer(const PVR_TIMER &timer)
   if (IsAutoTimer(timer))
     return DeleteAutoTimer(timer);
 
-  const std::string strServiceReference = m_channels.GetChannel(timer.iClientChannelUid)->GetServiceReference().c_str();
+  const auto it = std::find_if(m_timers.cbegin(), m_timers.cend(), [timer](const Timer& myTimer)
```
[the remark this answers is not in the document]
Comment (phunkyfish) [reply src/enigma2/Timers.cpp]: Done

File: src/enigma2/Timers.cpp (hunk 1 above)
Comment (phunkyfish) [reply src/enigma2/Timers.cpp]: Done

**Comment by phunkyfish** [comment]
Ready to go

**Review by ksooo — APPROVED** [review]
👍

**Pull request merged**